jib-core/src/test/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStepTest.java

```diff
@@ -87,6 +87,40 @@ public void setUp() {
     pullBaseImageStep = new PullBaseImageStep(buildContext, progressDispatcherFactory);
   }
 
+  @Test
+  public void testCall_scratch_noPlatformSpecified()
```

Comment:
Since the platform is actually specified as "slim arch" and "fat system",

```suggestion
  public void testCall_scratch_singlePlatform()
```